packages/notifications/src/styled/content/StyledClose.tsx

```diff
@@ -5,30 +5,51 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
+import React, { HTMLAttributes } from 'react';
 import styled from 'styled-components';
-import classNames from 'classnames';
-import CalloutStyles from '@zendeskgarden/css-callouts';
 import { retrieveComponentStyles } from '@zendeskgarden/react-theming';
+import XStrokeIcon from '@zendeskgarden/svg-icons/src/12/x-stroke.svg';
+import { IStyledAlertProps } from '../StyledAlert';
 
 const COMPONENT_ID = 'notifications.close';
 
-export interface IStyledCloseProps {
+export interface IStyledCloseProps extends IStyledAlertProps {
   isFocused?: boolean;
   isHovered?: boolean;
 }
 
+const Button = React.forwardRef<HTMLButtonElement, HTMLAttributes<HTMLButtonElement>>(
+  (props, ref) => (
+    <button ref={ref} {...props}>
+      <XStrokeIcon />
+    </button>
+  )
+);
+
 /**
  * Used to close a Notification. Supports all `<button>` props
+ *
+ * 1. Reset for <button> element.
  */
-export const StyledClose = styled.button.attrs<IStyledCloseProps>(props => ({
+export const StyledClose = styled(Button).attrs({
   'data-garden-id': COMPONENT_ID,
-  'data-garden-version': PACKAGE_VERSION,
-  type: 'button',
-  className: classNames(CalloutStyles['c-callout__close'], {
-    // State
-    [CalloutStyles['is-focused']]: props.isFocused,
-    [CalloutStyles['is-hovered']]: props.isHovered
-  })
-}))<IStyledCloseProps>`
+  'data-garden-version': PACKAGE_VERSION
+})<IStyledCloseProps>`
+  display: block;
+  position: absolute;
+  top: ${props => props.theme.space.base}px;
+  right: ${props => !props.theme.rtl && `${props.theme.space.base}px`};
+  left: ${props => props.theme.rtl && `${props.theme.space.base}px`};
+  transition: background-color 0.1s ease-in-out;
+  border: none; /* [1] */
+  border-radius: 50%;
+  background-color: transparent; /* [1] */
+  cursor: pointer;
+  padding: 0;
+  width: ${props => props.theme.space.base * 7}px;
+  height: ${props => props.theme.space.base * 7}px;
+  overflow: hidden;
+  font-size: 0; /* [1] */
```

Comment:
Shouldn't be needed as this `font-size` is a reset for `<a>` tags.

Reply:
My bad, the removal of this affects the alignment of the X within the button. It needs to be restored.